x-pack/plugins/actions/common/servicenow_config.ts

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
These definitions were originally in a server directory, and accessed from a public directory, with an eslint directive to not generate a warning about that.  However, when the actions plugin grew a public plugin, things got worse - we started getting a runtime message when Kibana started.  So I extracted all the stuff from the server directory that the public code needed, and moved it in here.